### PR TITLE
Update forging:enable command to accept maxHeightPreviouslyForged and force args - Closes #398

### LINKED ISSUE #398: Update forging:enable command to accept maxHeightPreviouslyForged and force args
### Description

- accept maxHeightPreviouslyForged and force parameters and call actions with those 2 arguments
- force should be optional and default false

### Motivation

- Enabling forging should be safe by default

### Acceptance Criteria

- It should work with issue https://github.com/LiskHQ/lisk-sdk/issues/5839
- Add unit test

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #398 

### How was it solved?

- Updated forging:enable command
- add two new args maxHeightPreviouslyForged and force
- Update unit test and add new tests

### How was it tested?

- Added unit tests
- Manual testing (build lisk-sdk from branch https://github.com/LiskHQ/lisk-sdk/pull/5871)
- Run `./bin/run forging:enable aebd99f07218109162a905d0e0c91e58bedc83c5 0 0 0 true`

### CODE REVIEW

File: src/base_forging.ts [hunk 1]
```diff
@@ -21,8 +21,14 @@ import BaseIPCCommand from './base_ipc';
 
 interface Args {
 	readonly address: string;
+	readonly height: number;
+	readonly maxHeightPreviouslyForged: number;
+	readonly maxHeightPrevoted: number;
+	readonly overwrite: boolean;
```
Comment (shuse2) [inline src/base_forging.ts:27]: overwrite should be flag as it's optional. Also, height, maxHeightPreviouslyForged and maxHeightPrevoted only exists in the enable command?

**Review by shuse2 — COMMENTED** [review]
test has conflict now 🙏

File: src/base_forging.ts [hunk 2]
```diff
@@ -21,8 +21,15 @@ import BaseIPCCommand from './base_ipc';
 
 interface Args {
 	readonly address: string;
+	readonly height?: number;
+	readonly maxHeightPreviouslyForged?: number;
+	readonly maxHeightPrevoted?: number;
+	readonly overwrite?: boolean;
```
Comment (shuse2) [inline src/base_forging.ts:27]: overwrite should be flag, and default to false

**Review by shuse2 — CHANGES_REQUESTED** [review]
mostly looks good to me, just the arg vs flag

**Review by shuse2 — APPROVED** [review]

**Review by ishantiw — APPROVED** [review]

**Pull request merged**